pontoon/sync/tests/formats/__init__.py

```diff
@@ -246,6 +246,26 @@ def run_parse_empty_translation(self, input_string, translation_index):
                 source_string='Empty Translation',
             )
 
+    def run_parse_context(self, input_string, translation_index):
```

Comment:
If this test is pofile-specific, then it can just go in `test_po.py`. Do any other formats support context in this way?